scio-core/src/main/scala/com/spotify/scio/transforms/package.scala

```diff
@@ -71,6 +71,78 @@ package object transforms {
 
   }
 
+  implicit class LimitedParallelismDoFns[T: ClassTag](val self: SCollection[T]) {
```

Comment:
Scaladoc with a brief description like the `PipeSCollection` below? It should also `extend AnyVal` and probably be named `LimitedParallelismSCollection`?

Reply:
FYI it's available via `self` so you can get rid of `ClassTag` context bound and add `implicit val ct = self.ct` inside the code.

Reply:
Ah didn't know that 👏